pyramid_learning_journal/pyramid_learning_journal/views/default.py

```diff
@@ -1,43 +1,29 @@
-    {'id': 7, 'title': 'Week 1: Day 5', 'date': 'May 19, 2017', 'body': """Today was really great practice. I wasn't terribly challenged by the katas I did, perhaps I should have tried more difficult ones, but I got so much practice with the python syntax and core methods. Writing all the tests wasn't fun but it was great to get into the flow of running the tests over and over. I wasn't looking forward to today when it was announced what we would be doing, but I really think I got a lot out of it and I'm glad we did it."""},
-    {'id': 8, 'title': 'Week 1: Day 4', 'date': 'May 18, 2017', 'body': """Today I learned about creating a tox file and how to use it for testing. The lecture reinforced some understanding of lists, strings, tuples, and dictionaries. It also introduced new ideas, including using lambda functions to create small anonymous functions."""},
-    {'id': 9, 'title': 'Week 1: Day 3', 'date': 'May 17, 2017', 'body': """Today I learned how to set up the setup.py file and how to use it. I also learned what 401 Python actually feels like. At least I think I did. I put in several extra hours and it still wasn't enough. Today's lab was rough, but we got super close to completing it."""},
-    {'id': 10, 'title': 'Week 1: Day 2', 'date': 'May 16, 2017', 'body': """Today I learned a lot more about parameters. I wrote and ran the first few tests of my programing life. Using the local environment and running code in the command line became a little bit more clear to me."""},
-    {'id': 11, 'title': 'Week 1: Day 1', 'date': 'May 15, 2017', 'body': """Today I learned that flow of Python is going to be a lot different from Javascript. I'm completely unfamiliar with the way we used the command line today and following along was difficult. I probably should have asked some questions but I wasn't sure where to even begin. Excited to figure it all out."""}
-
-]
+from pyramid_learning_journal.models import Journal
 
 
 @view_config(route_name='home', renderer='../templates/index.jinja2')
 def home_view(request):
     """View for the home route."""
+    session = request.dbsession
+    all_journals = session.query(Journal).all()
     return {
         'page': 'Home',
-        'journals': JOURNALS
+        'journals': all_journals
     }
 
 
 @view_config(route_name='detail', renderer="../templates/individual-entry.jinja2")
 def detail_view(request):
     """View for the journal route."""
     the_id = int(request.matchdict['id'])
-    journal = None
-    try:
-        journal = JOURNALS[the_id]
-    except IndexError:
+    session = request.dbsession
+    entry = session.query(Journal).get(the_id)
+    if not entry:
         raise HTTPNotFound
 
     return {
         'page': 'Journal Entry',
-        'journal': journal
+        'entry': entry
     }
```

Comment:
You should also have add and edit views, even if they just return an empty dict.